src/commands/synthetics/interfaces.ts

```diff
@@ -267,3 +267,18 @@ export interface APIConfiguration {
   baseUrl: string
   proxyOpts: ProxyConfiguration
 }
+
+export interface CommandConfig {
+  apiKey: string
+  appKey: string
+  configPath: string
+  datadogSite: string
+  fileGlobs: string[]
```

Comment:
We should keep the same name as before, aka `files`, as it is documented [here](https://github.com/DataDog/datadog-ci/tree/master/src/commands/synthetics#configuration) and [here](https://docs.datadoghq.com/synthetics/ci/?tab=apitest#setup-the-client).
`fileGlobs` was the CLI parameter.